test/e2e/operator.go

```
@@ -329,6 +351,49 @@ func TestKCMRecovery(t *testing.T) {
+	kubeConfig, err := testlib.NewClientConfigForTest()
+	require.NoError(t, err)
+	kubeClient, err := kubernetes.NewForConfig(kubeConfig)
+	require.NoError(t, err)
+	operatorClientSet, err := operatorv1client.NewForConfig(kubeConfig)
+	require.NoError(t, err)
+	kcmOperator := operatorClientSet.KubeControllerManagers()
+	kcmOperatorConfig, err := kcmOperator.Get(context.TODO(), "cluster", metav1.GetOptions{})
+	require.NoError(t, err)
+
+	kcmOperatorConfig.Spec.LogLevel = "Debug"
+	kcmOperator.Update(context.TODO(), kcmOperatorConfig, metav1.UpdateOptions{})
```

golangci-lint (2.12.1)

[error] Error return value of `kcmOperator.Update` is not checked

(errcheck)